modules/machineset-yaml-nutanix.adoc

```diff
@@ -78,36 +91,37 @@ endif::infra[]
       providerSpec:
         value:
           apiVersion: machine.openshift.io/v1
-          cluster:
+          categories: <5>
+          - key: <category_name>
+            value: <category_value>
+          cluster: <6>
             type: uuid
             uuid: <cluster_uuid>
           credentialsSecret:
             name: nutanix-creds-secret
           image:
-            name: <infrastructure_id>-rhcos <5>
+            name: <infrastructure_id>-rhcos <7>
             type: name
           kind: NutanixMachineProviderConfig
-          memorySize: 16Gi <6>
+          memorySize: 16Gi <8>
+          project: <9>
+            type: name
+            uuid: <project_name>
```

Comment:
Seems `uuid` should be changed to `name` here.
`type` can be `name` or `uuid`, and they should be consistent. for example:
```
          project:
            type: name
            name: <project_name>
```
or
```
          project:
            type: uuid
            uuid: <project_uuid>
```